src/lib/components/chip/index.tsx

```diff
@@ -25,16 +25,14 @@ export default ({
       className={`c-pointer ${styles['chip-button-container']}`}
       onClick={() => onRemove(value)}
     >
-      <img
-        className={styles['chip-remove-button-highlighted']}
-        src={removeButtonHighlightedIcon}
-        alt="removal x button highlighted"
-      />
-      <img
-        className={styles['chip-remove-button']}
-        src={removeButtonIcon}
-        alt="removal x button"
-      />
+      <XIcon color={'purple-300'} className={styles['chip-remove-button']} />
+      {
```

Comment:
I think this `{` is not needed.

I think this could also be changed to have just on `XIcon` and remove the `chip-remove-button-highlighted` className. Something like:

```
 <div
      className={`c-pointer ${styles['chip-button-container']}`}
      onClick={() => onRemove(value)}
    >
      <XIcon className={styles['chip-remove-button']}  />
      </div>
```

```
.chip-button-container {
  color: $ds-purple-300;
  
  &:hover {
    color: $ds-purple-500;
    }
}
```

Reply:
Also check this XIcon, it seems to not be vertically centred anymore

Reply:
@diogomateus That makes sense, and you are right: the `XIcon` was falling below the center. It took me a while to fix it, and it only worked when I got to styling the SVG itself. Not sure if there's a simpler fix I missed. Anyway, I worked on the changes you suggested both here and in `informationBox`!